### PR TITLE
Add rjsmin

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/rjsmin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (synapticarbors) [inline recipes/rjsmin/meta.yaml]: It looks like this is building a C-extension, so please add `toolchain` to the `build` section.

File: recipes/rjsmin/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - rjsmin
```
Comment (synapticarbors) [inline recipes/rjsmin/meta.yaml]: Could you also add `_rjsmin` to the `imports` test to try to import the compiled c-extension.

**Comment by sannykr** [comment]
@synapticarbors : Hi Joshua, this package is available on conda-forge, but the build is not successful for OSX python 3.6.    Is it possible to trigger the build again? 

Regards
Sanny

**Comment by synapticarbors** [comment]
Can you ask on the gitter group? https://gitter.im/conda-forge/conda-forge.github.io I'm not available to do this right now.

**Comment by sannykr** [comment]
Thanks. I will do that.

**Pull request merged**